core/win32/syscall.c

```diff
@@ -4288,6 +4367,9 @@ void post_system_call(dcontext_t *dcontext)
         postsys_MapViewOfSection(dcontext, param_base, success);
         KSTOP(post_syscall_map);
```

Comment:
We probably want the same KSTART..KSTOP for Ex but we could move it to some shared handler once we have Ex handling: they should share a lot of code.

Reply:
Yeah, I was wondering about those. I'll add them in the subsequent PR if the handlers are too different (but they shouldn't be, as you said).